tests/MarginTrading.SettingsService.TestClient/Program.cs

```diff
@@ -0,0 +1,275 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AsyncFriendlyStackTrace;
+using JetBrains.Annotations;
+using Lykke.HttpClientGenerator;
+using MarginTrading.SettingsService.Client;
+using MarginTrading.SettingsService.Client.Asset;
+using MarginTrading.SettingsService.Client.AssetPair;
+using MarginTrading.SettingsService.Client.Enums;
+using MarginTrading.SettingsService.Client.Market;
+using MarginTrading.SettingsService.Client.Routes;
+using MarginTrading.SettingsService.Client.Scheduling;
+using MarginTrading.SettingsService.Client.TradingConditions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Newtonsoft.Json;
+using Refit;
+
+namespace MarginTrading.SettingsService.TestClient
+{
+    /// <summary>
+    /// Simple way to check api clients are working.
+    /// In future this could be turned into a functional testing app.
+    /// </summary>
+    internal static class Program
+    {
+        private static int _counter;
+
+        static async Task Main()
+        {
+            try
+            {
+                await Run();
+            }
+            catch (ApiException e)
+            {
+                var str = e.Content;
+                if (str.StartsWith('"'))
+                {
+                    str = TryDeserializeToString(str);
+                }
+
+                Console.WriteLine(str);
+                Console.WriteLine(e.ToAsyncString());
+            }
+        }
+
+        private static string TryDeserializeToString(string str)
+        {
+            try
+            {
+                return JsonConvert.DeserializeObject<string>(str);
+            }
+            catch
+            {
+                return str;
+            }
+        }
+
+        private static async Task Run()
+        {
+            var clientGenerator = HttpClientGenerator.BuildForUrl("http://localhost:5000").Create();
```

Comment:
TestClient checks only that API works, but not that it works correctly )

Reply:
Added value checks

Reply:
Will be impl in a separate ticket